### PR TITLE
Admin Trace UI - Revival

### PR DESCRIPTION
**Pull request by dvaldivia**

### CODE REVIEW

File: portal-ui/src/screens/Console/Trace/Trace.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        m.time = new Date(m.time.toString());
+        m.key = Math.random();
+        traceMessageReceived(m);
+      };
+      c.onclose = () => {
+        clearInterval(interval);
+        console.log("connection closed by server");
+      };
+      return () => {
+        c.close();
```
Comment (cesnietor) [inline portal-ui/src/screens/Console/Trace/Trace.tsx:100]: could you please send a a valid closing code? else it is `close 1005 (no status)` 
https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent#Status_codes

**Comment by cesnietor** [comment]
the assets seem not to be built

File: portal-ui/src/screens/Console/Trace/Trace.tsx [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+    //less than ten of KB or greater units
+    return n.toFixed(n < 10 && l > 0 ? 1 : 0) + " " + units[l];
+  };
+  const timeFromdate = (d: Date) => {
+    let h = d.getHours() < 10 ? `0${d.getHours()}` : `${d.getHours()}`;
+    let m = d.getMinutes() < 10 ? `0${d.getMinutes()}` : `${d.getMinutes()}`;
+    let s = d.getSeconds() < 10 ? `0${d.getSeconds()}` : `${d.getSeconds()}`;
+
+    return `${h}:${m}:${s}:${d.getMilliseconds()}`;
+  };
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Trace/Trace.tsx:125]: This can be placed in utils file

File: portal-ui/src/screens/Console/Trace/Trace.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+  messages
+}: ITrace) => {
+  useEffect(() => {
+    traceResetMessages();
+    const token: string = storage.getItem("token")!;
+    const url = new URL(window.location.toString());
+    const isDev = process.env.NODE_ENV === "development";
+    const port = isDev ? "9090" : url.port;
+
+    const setCookie = (name: string, val: string) => {
```
Comment (bexsoft) [inline portal-ui/src/screens/Console/Trace/Trace.tsx:64]: This can be set in utils file too

**Pull request merged**